grpc-server-security-jwt/src/main/java/io/micronaut/grpc/server/security/jwt/GrpcServerSecurityJwtConfiguration.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.grpc.server.security.jwt;
+
+import edu.umd.cs.findbugs.annotations.Nullable;
+import io.grpc.Status;
+import io.micronaut.context.annotation.ConfigurationProperties;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.core.bind.annotation.Bindable;
+import io.micronaut.core.order.Ordered;
+import io.micronaut.core.util.StringUtils;
+import io.micronaut.core.util.Toggleable;
+import io.micronaut.grpc.server.GrpcServerConfiguration;
+import io.micronaut.security.config.InterceptUrlMapPattern;
+import io.micronaut.security.config.SecurityConfigurationProperties;
+import io.micronaut.security.token.config.TokenConfigurationProperties;
+import io.micronaut.security.token.jwt.config.JwtConfigurationProperties;
+
+import javax.validation.constraints.NotBlank;
+import java.util.Collection;
+
+/**
+ * gRPC Security JWT configuration.
+ *
+ * @since 2.4.0
+ * @author Brian Wyka
+ */
+@ConfigurationProperties(GrpcServerSecurityJwtConfiguration.PREFIX)
+@Requires(property = SecurityConfigurationProperties.PREFIX + ".enabled", notEquals = StringUtils.FALSE)
+@Requires(property = TokenConfigurationProperties.PREFIX + ".enabled", notEquals = StringUtils.FALSE)
+@Requires(property = JwtConfigurationProperties.PREFIX + ".enabled", notEquals = StringUtils.FALSE)
+@Requires(property = GrpcServerSecurityJwtConfiguration.PREFIX + ".enabled", notEquals = StringUtils.FALSE)
```

Comment:
This is not so great since it has to duplicate the knowledge about defaults and also what the keys are

Instead you can do `@Requires(configuration = 'io.micronaut.security.token.jwt')` and `@Requires(configuration = 'io.micronaut.security')`

Reply:
Sure.  I can make that change.

Reply:
Done.